src/modules/login/index.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+
+import { TouchableOpacity, StyleSheet, TextInput } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import { Ionicons } from '@expo/vector-icons';
+
+import { Text, View } from '../../components/Themed';
+
+
+export default function LoginScreen() {
+
+    return (
+        <View style={styles.container}>
+            <LinearGradient colors={['#051937', '#004d7a', '#008793', '#00bf72', '#a8eb12']} style={styles.header}>
+                <View style={styles.logo}>
+                    <View style={styles.image}></View>
+                </View>
+
+                <Text style={[styles.text, styles.title]}>Login</Text>
+
+            </LinearGradient>
+            <View style={styles.main}>
+                <View style={styles.form}>
+                    <Ionicons style={styles.iconEmail} size={25} name="mail" />
+                    <TextInput
+                        keyboardType={'email-address'}
+                        style={styles.input}
+                        placeholder="Email"
+                    />
+
+                    <Ionicons style={styles.iconPassword} size={25} name="key" />
+                    <TextInput
+                        style={styles.input}
+                        placeholder="Password"
+                    />
+                    <TouchableOpacity>
+                        <LinearGradient colors={['#051937', '#004d7a', '#008793', '#00bf72', '#a8eb12']} style={styles.button}>
+                            <Text style={styles.text}>LOGIN</Text>
+                        </LinearGradient>
+                    </TouchableOpacity>
+                </View>
+            </View>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
```

Comment:
To avoid SUPER CLASS, move common components and their logic and styles to separate files.